types/youtube/index.d.ts

```diff
@@ -13,6 +13,10 @@
  * @see https://developers.google.com/youtube/iframe_api_reference
  * @see https://developers.google.com/youtube/player_parameters
  */
+interface Window {
+    onYouTubeIframeAPIReady(): void;
```

Comment:
Since it probably won't be defined until it is set, I would suggest this:
```suggestion
    onYouTubeIframeAPIReady?(): void;
```

Reply:
Thanks for the review. imo, that should not be optional as per [Google doc]:(https://developers.google.com/youtube/iframe_api_reference#Requirements)

> Any web page that uses the IFrame API **must** also implement the following JavaScript function:
>
> onYouTubeIframeAPIReady – The API will call this function when the page has finished downloading the JavaScript for the player API, which enables you to then use the API on your page. Thus, this function might create the player objects that you want to display when the page loads.

A similar thing is done at `@types/spotify-web-playback-sdk`:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/spotify-web-playback-sdk/index.d.ts#L12

Either way though, the way we define it does not error the build if it is not set.